src/canopy/chat_engine/chat_engine.py

```diff
@@ -227,9 +229,10 @@ def chat(self,
 
     def _get_context(self,
                      messages: Messages,
+                     namespace: Optional[str] = None
                      ) -> Context:
         queries = self._query_builder.generate(messages, self.max_prompt_tokens)
-        context = self.context_engine.query(queries, self.max_context_tokens)
+        context = self.context_engine.query(queries, self.max_context_tokens, namespace)
```

Comment:
```suggestion
        context = self.context_engine.query(queries, self.max_context_tokens, namespace=namespace)
```
We're trying to be very explicit about keyword args, and rely on argument ordering only 1-2 mandatory args.